pages/docs/community/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Overview
+weight: 2
+---
+
+## Community: Guidelines and resources around community.
+
+Welcome to AsyncAPI **Community**! Our Community section documents the community guidelines and resources.
+
+<Remember>
+
+## Contribute to the AsyncAPI Community section
+Code isn't the only way to contribute to OSS; Dev Docs are a **huge** help that benefit the entire OSS ecosystem. At AsyncAPI, we value Doc contributions as much as every other type of contribution. ❤️
+
+To get started as a Docs contributor:
+1. Familiarize yourself with our [project's Contribution Guide](https://github.com/asyncapi/community/blob/master/CONTRIBUTING.md) and our [Code of Conduct](https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md).
+2. Head over to our [AsyncAPI Docs Board](https://github.com/orgs/asyncapi/projects/8).
```

Comment:
```suggestion
2. Head over to our [AsyncAPI Docs Board](https://github.com/orgs/asyncapi/projects/12).
```

it's actually changed because when I migrated the docs board to the latest GH offered, the URL updated too